bin/BuildPackages.sh

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env bash
 
+LOGDIR=log
+if [ ! -d "$LOGDIR" ]
+  then
+  mkdir -p $LOGDIR
+fi
+LOGFILE=buildpackages
+FAILPKGFILE=fail
+
+build_all_packages() {
+
```

Comment:
This change breaks handling of the argument. I have my `pkg` outside the repository, so that I can share it with multiple conccurrent builds. I invoke the script like this:
```
$GAPROOT/bin/BuildPackages.sh $GAPROOT
```
This is broken with this PR. You need to parse arguments to the script first. Then, pass GAPDIR as a mandatory argument to `build_all_packages`.